lib/dialyzer/src/dialyzer_analysis_callgraph.erl

```diff
@@ -117,25 +117,16 @@ loop(#server_state{parent = Parent} = State,
 analysis_start(Parent, Analysis, LegalWarnings) ->
   CServer = dialyzer_codeserver:new(),
   Plt = Analysis#analysis.plt,
-  State = #analysis_state{codeserver = CServer,
```

Comment:
this is the removal of tabs and replace of this record creation into its own function